automation_tools/repository.py

```diff
@@ -99,15 +99,22 @@ def create_custom_repos(**kwargs):
         repo_file.close()
 
 
-def enable_satellite_repos(cdn=False, disable_enabled=True):
+def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True):
```

Comment:
Will adding a new argument in this manner break any calls to this method that use positional arguments? Or do we have no calls to this method that are positional?